src/main/kotlin/no/nav/familie/tilbake/api/DokumentRestTjeneste.kt

```diff
@@ -0,0 +1,27 @@
+package no.nav.familie.tilbake.api
+
+import no.nav.familie.tilbake.service.dokumentbestilling.brevmaler.Dokumentmalstype
+import no.nav.familie.tilbake.service.dokumentbestilling.varsel.VarselbrevTjeneste
+import no.nav.familie.tilbake.service.dokumentbestilling.varsel.manuelt.ManueltVarselBrevTjeneste
+import no.nav.familie.tilbake.sikkerhet.Behandlerrolle
+import no.nav.familie.tilbake.sikkerhet.Rolletilgangssjekk
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.http.MediaType
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.util.UUID
+
+@RestController
+@RequestMapping("/api/dokument")
+@ProtectedWithClaims(issuer = "azuread")
+class DokumentRestTjeneste(val varselbrevTjeneste: VarselbrevTjeneste,
+                           val manueltVarselBrevTjeneste: ManueltVarselBrevTjeneste) {
+
+    @GetMapping("/forhandsvis-varselbrev",
```

Comment:
Lurer på om denne bør renames til berre "/api/dokument/forhandsvis". Tror det er dette endepunktet som vil bli brukt av "Send melding"-delen i frontend, sidan den får maltype og fritekst.
Denne vil også måtte støtte "Innhent dokumentasjon" på eit seinare tidspunkt.

Den kan vel nesten støtte den malen allereie? Innhent dokumentasjon er jo nesten ferdigimplementert.

Det vil være måtte være eit anna endepunkt for forhåndsvisning av Varselbrev frå simuleringsbildet i fagsystema.